dotty/matchers-core/src/main/scala/org/scalatest/matchers/dsl/BeWord.scala

```diff
@@ -90,7 +90,7 @@ final class BeWord {
    *                       ^
    * </pre>
    **/
-  def <[T : Ordering](right: T): Matcher[T] =
+  infix def <[T : Ordering](right: T): Matcher[T] =
     new Matcher[T] {
```

Comment:
I would remove infix from methods with symbolic names, because it is not required for symbols.